tests/test_extensions/test_sklearn_extension/test_sklearn_extension.py

```diff
@@ -2253,3 +2253,15 @@ def column_transformer_pipe(task_id):
         run2 = column_transformer_pipe(23)  # only numeric
         TestBase._mark_entity_for_removal("run", run2.run_id)
         self.assertEqual(run1.setup_id, run2.setup_id)
+
+    def test_for_refit_time_in_basesearchCV(self):
+        X, y = sklearn.datasets.load_iris(return_X_y=True)
+        rs = sklearn.model_selection.GridSearchCV(
+            estimator=sklearn.ensemble.RandomForestClassifier(),
+            param_grid={"n_estimators": [2, 3, 4, 5]},
+        )
+        rs.fit(X, y)
```

Comment:
Could you please try to integrate this test into an existing unit test? Right now this only tests scikit-learn, which I don't think we have to do.